control-plane/msp-operator/src/main.rs

```diff
@@ -479,8 +479,16 @@ impl ResourceContext {
             });
         }
 
-        let mut body = HashMap::new();
-        body.insert("disks", self.spec.disks.clone());
+        let mut labels: HashMap<String, String> = HashMap::new();
+        labels.insert(
+            String::from("openebs.io/created-by"),
+            String::from("mayastor-control-plane"),
```

Comment:
I wonder if this should be "msp-operator" as that is what is requesting that a pool should be created.